### PR TITLE
Population type to show only KP type. Formatted phone numbers, Disabi…

### PR DESCRIPTION
**Pull request by njorocs**
…lity column only to show disability type, Added Linked column to HTS register, mapped marital status for never married and single correctly, added encounter date

### CODE REVIEW

File: omod/src/main/webapp/fragments/patient/editPatient.gsp [hunk 1]
```diff
@@ -313,7 +313,7 @@
 
     <div class="ke-panel-footer">
         <button type="submit">
-            <img src="${ui.resourceLink("kenyaui", "images/glyphs/ok.png")}"/> ${command.original ? "Save Changes" : "Create Patient"}
+            <img src="${ui.resourceLink("kenyaui", "images/glyphs/ok.png")}"/> ${command.original ? "Save Changesma" : "Create Patient"}
```
Comment (ojwanganto) [inline omod/src/main/webapp/fragments/patient/editPatient.gsp:316]: please revert this.

File: api/src/main/resources/messages.properties [hunk 2]
```diff
@@ -72,7 +72,8 @@ ui.i18n.Concept.name.${metadata.concept.MARRIED_MONOGAMOUS}=Married monogamous
 ui.i18n.Concept.name.${metadata.concept.MARRIED_POLYGAMOUS}=Married polygamous
 ui.i18n.Concept.name.${metadata.concept.NEGATIVE}=Negative
 ui.i18n.Concept.name.${metadata.concept.PRIMARY_EDUCATION}=Primary school education
-ui.i18n.Concept.name.${metadata.concept.NEVER_MARRIED}=Single
+ui.i18n.Concept.name.${metadata.concept.NEVER_MARRIED}=Never married
+ui.i18n.Concept.name.${metadata.concept.LIVING_ALONE}=Single
```
Comment (ojwanganto) [inline api/src/main/resources/messages.properties:76]: How did we used to handle this before? Didn't we have an option for single?

File: api/src/main/java/org/openmrs/module/kenyaemr/reporting/builder/hts/HTSRegisterReportBuilder.java [hunk 3]
```diff
@@ -137,6 +118,7 @@ protected DataSetDefinition datasetColumns() {
         dsd.addColumn("tbScreening", new HTSTBScreeningDataDefinition(), null);
         dsd.addColumn("everHadHIVSelfTest", new HTSSelfTestDataDefinition(), null);
         dsd.addColumn("assessedForHIVRisk", new HTSRiskDoneDataDefinition(),"", new HTSRiskAssessedConverter());
+        dsd.addColumn("Linked", new HTSLinkageIdentifierDataDefinition(), null);
```
Comment (ojwanganto) [inline api/src/main/java/org/openmrs/module/kenyaemr/reporting/builder/hts/HTSRegisterReportBuilder.java:121]: Does this also pick CCC number from person identifier? I can't remember clearly.

**Review by ojwanganto — CHANGES_REQUESTED** [review]
Please also just check that for phone numbers and date columns show correctly and in the right format

File: api/src/main/java/org/openmrs/module/kenyaemr/reporting/builder/hts/HTSRegisterReportBuilder.java (hunk 3 above)
Comment (njorocs) [reply api/src/main/java/org/openmrs/module/kenyaemr/reporting/builder/hts/HTSRegisterReportBuilder.java:121]: The query is getting ccc number from  a coalesce of (unique_patient_no from demographics, ccc_number from etl_hts_referral_and_linkage) .

**Review by patryllus — APPROVED** [review]

**Review by patryllus — APPROVED** [review]

**Pull request merged**